server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -240,10 +240,14 @@ protected void mirrorRemoteToLocal(File workDir, CommandExecutor executor,
                 // Add mirror_state.json.
                 changes.put(mirrorStatePath, Change.ofJsonUpsert(
                         mirrorStatePath, "{ \"sourceRevision\": \"" + id.name() + "\" }"));
-
                 // Construct the log message and log.
-                summary = "Mirror " + abbrId + ", " + remoteRepoUri() + '#' + remoteBranch() +
-                          " to the repository '" + localRepo().name() + '\'';
+                final String headCommitMessage = revWalk.parseCommit(id).getFullMessage();
+                if (!headCommitMessage.isEmpty()) {
+                    summary = headCommitMessage;
+                } else {
+                    summary = "Mirror " + abbrId + ", " + remoteRepoUri() + '#' + remoteBranch() +
+                              " to the repository '" + localRepo().name() + '\'';
```

Comment:
What do you thinking of fully appending the upstream's commit message to the detail of the mirroring summary. For example
```
Mirror abbrId, remote to the repository local

The commit information of 'abbrId':
Author:  John Doe <johne.doe@gmail.com>
Date:    Thu Mar 2 19:43:28 2023 +0900
Message: Set xxx to yyy to enable zzz feature ...
```

Reply:
Thanks for the suggestion. 
![image](https://user-images.githubusercontent.com/11583408/223339494-a3fdf67b-d082-447a-adff-bbbf1d8ff743.png)
As I understand (correct me if I am wrong), we change `2` to meaning message, from my side, I think the full message is enough.
How do you think @trustin @jrhee17 @minwoox

Reply:
The UI for commit messages will be improved to show the message in detail. REST API could be used to get the commit message of a revision.

Reply:
I think we don't need to change the summary but just add the author, date and commit message from the remote into the detail.

Reply:
As you might have noticed, Central Dogma's `Commit` object has `summary` and `detail` field. Generate the string @ikhoon suggested and put it into `Commit.detail`.

Reply:
Seem I misunderstand the requirement, let me check, thank you.

Reply:
Added info to detail field